atlassian_jwt_auth/contrib/tests/utils.py

```diff
@@ -1,20 +1,21 @@
 import atlassian_jwt_auth
 
 
-class StaticPublicKeyRetriever(object):
```

Comment:
I think this is being a bit overly clever and that a simpler and better solution is to have the flask `requires_asap` decorator work with a `verifier` keyword argument - does that make sense? (my memory of decorators is a little bit rusty)